xml/mdns_device-info_txt.xml

```diff
@@ -342,6 +354,18 @@
     <param pos="0" name="hw.product" value="MacBook (Retina, 12-inch, 2017)"/>
     <param pos="0" name="hw.device" value="Laptop"/>
   </fingerprint>
+  <fingerprint pattern="^model=MacBookPro9,2$">
+    <description>MacBook Pro (13-inch, Mid 2012)</description>
+    <example>model=MacBookPro9,2</example>
+    <param pos="0" name="os.vendor" value="Apple"/>
+    <param pos="0" name="os.cpe23" value="cpe:/o:apple:mac_os_x:-"/>
+    <param pos="0" name="os.family" value="Mac OS X"/>
+    <param pos="0" name="os.product" value="Mac OS X"/>
+    <param pos="0" name="hw.vendor" value="Apple"/>
+    <param pos="0" name="hw.family" value="MacBook"/>
+    <param pos="0" name="hw.product" value="MacBook (13-inch, Mid 2012)"/>
```

Comment:
`MacBookPro9,2` is a McBook Pro which should be reflected in the family and product attributes.